CHANGELOG_DESKTOP.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## [1.42.86](https://github.com/brave/brave-browser/releases/tag/v1.42.86)
+
+ - Added Aurora EVM chain to preloaded chains for Brave Wallet. ([#23522](https://github.com/brave/brave-browser/issues/23522))
+ - Added loading skeletons to Brave Wallet panel. ([#23577](https://github.com/brave/brave-browser/issues/23577))
+ - Added support for using Ledger hardware for Solana signTransaction, signAllTransactions provider APIs with Brave Wallet. ([#23842](https://github.com/brave/brave-browser/issues/23842))
+ - Added a list of well known chains to the local data files updater for Brave Wallet. ([#14574](https://github.com/brave/brave-browser/issues/14574))
+ - Added a link to the Brave Help Center from various places within Brave Wallet. ([#22598](https://github.com/brave/brave-browser/issues/22598))
+ - Added the ability to import from the Chromium version of Microsoft Edge. ([#13334](https://github.com/brave/brave-browser/issues/13334))
+ - Added setting for search engines used in the address bar in private window under brave://settings/search. ([#23091](https://github.com/brave/brave-browser/issues/23091))
+ - Added settings to refine autocomplete suggestions in the address bar under brave://settings/appearance. ([#22948](https://github.com/brave/brave-browser/issues/22948))
+ - Added the ability to debounce URLs based on a regex applied to the path. ([#23121](https://github.com/brave/brave-browser/issues/23121))
+ - [Security] Added a warning to indicate that local IPFS node will bypass browser proxy as reported on HackerOne by neeythann. ([#23570](https://github.com/brave/brave-browser/issues/23570))
+ - [Security] Fixed IPFS crash as reported on HackerOne by neeythann. ([#23646](https://github.com/brave/brave-browser/issues/23646))
+ - [Security] Fixed wallet API freezing issue as reported on HackerOne by jordaaash. ([#23973](https://github.com/brave/brave-browser/issues/23973))
+ - Enabled "unlinkable bouncing" by default. ([#23180](https://github.com/brave/brave-browser/issues/23180))
+ - Improved fingerprint farbling in aggressive mode. ([#24126](https://github.com/brave/brave-browser/issues/24126))
+ - Updated default search engine to Brave Search for new installations in certain regions. ([#23814](https://github.com/brave/brave-browser/issues/23814))
+ - Updated macOS minimum version to 10.13 in Sparkle. ([#23748](https://github.com/brave/brave-browser/issues/23748))
+ - Updated New Tab page design for private window and private window with Tor. ([#22566](https://github.com/brave/brave-browser/issues/22566))
+ - Updated brave://settings/cookies to better align with brave://shields. ([#8604](https://github.com/brave/brave-browser/issues/8604))
+ - Updated eth_call in Brave Wallet to handle extraneous data correctly. ([#23998](https://github.com/brave/brave-browser/issues/23998))
```

Comment:
Probably change to `Fixed incorrect balance being shown for certain tokens`.

Reply:
Updated and added "in Brave Wallet" to the end of suggested text.